monasca/README.md

```diff
@@ -50,6 +62,19 @@ $ helm delete my-release --purge
 The command removes all the Kubernetes components associated with the chart and
 deletes the release.
 
+### Default monitoring
+
+By default Monasca will monitor pod workloads (CPU, Network, Memory, etc.) and Kubernetes health.
+
+It will also autodetect Prometheus Endpoints by looking for the following annotations on services and pods
+
+* prometheus.io/scrape: Only scrape pods that have a value of 'true'
+* prometheus.io/path: If the metrics path is not '/metrics' override this.
+* prometheus.io/port: Scrape the pod on the indicated port instead of the default of '9102'.
+
+More information on our monitoring within in Kubernetes can be found on
+[monasca.io Kubernetes](http://monasca.io/docs/kubernetes.html)
```

Comment:
Maybe be best to refer to it as 'the Monasca documentation' or just 'monasca.io' rather than 'monasca.io Kubernetes'